README.md

```diff
@@ -44,6 +44,14 @@ In order to make it work with the webpack-dev-server launched by the `npm run se
 cd ./node_modules/esn-frontend-login && npm install && npm run build
 ```
 
+### Serve
+
+After merging PRs related to docker-compose task, we cannot go to http://localhost:9900 after `npm run serve` for each SPA anymore.
+
+Instead, we need to specify the corresponding path for each SPA ([same as we did for dev server](https://ci.linagora.com/linagora/lgs/openpaas/esn/-/wikis/Frontend-Split/Continuous-Depoyment-on-dev))
+
+For this module, you need to go to: http://localhost:9900/contacts/
```

Comment:
I think we should not explain what we did not document before. Just a single line is enough

`Go to http://localhost:9900/contacts/ to open SPA`